doc/manual/documentation.rst

```diff
@@ -99,14 +99,17 @@ documentation:
        ...
        """
 
-5. Group examples under an ``# Examples`` section and use `````julia`` blocks instead of
-   standard text.
+5. Group examples under an ``# Examples`` section and use Markdown `Code blocks`_ instead of
```

Comment:
We should also mention that doctests shouldn't use functions like `rand` & friends.